tools/wptrunner/wptrunner/browsers/safari.py

```diff
@@ -2,8 +2,8 @@
 
 import os
 import plistlib
+from packaging.version import Version
```

Comment:
Can we make `packaging` an explicit requirement if we're going to start depending on it directly?